subhub/hub/verifications/events_check.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from abc import ABC
+import time
+from datetime import datetime, timedelta
+
+from subhub.app import create_app, g
+from subhub.hub.stripe.controller import event_process
+from flask import current_app
+import stripe
+
+from subhub.cfg import CFG
+from subhub.log import get_logger
+
+logger = get_logger()
+
+try:
+    app = create_app()
```

Comment:
Did you want to instrument this with XRay while at it?  You can just crib the other handler's middleware invocation.

Reply:
Yes, now that I have everything working I can add XRay.